source-git-monorepos/README.md

```diff
@@ -1,41 +1,80 @@
-# Support monorepos in source-git
+# Support for Monorepos
 
-Monorepos store the source code of multiple projects in a single
-repository. They are a special use-case from a source-git point of view,
-because an upstream monorepo maps to multiple dist-git repositories in the
-distribution.
+Monorepos store the source code of multiple projects in a single repository.
```

Comment:
I wonder if want to account for different build configurations too, e.g. have different compiler flags for multiple sets of builds.

Reply:
I would expect to find things related to build configuration in the spec-file of each package (or some other scripting called during *build-time*), and from this point of view I would say these fall outside the scope of source-git. Or am I missing something?

Reply:
I am thinking out loud here as I see a big potential in the monorepo concept that it could solve more use cases related to building.

rpmbuild and mock [support](https://github.com/rpm-software-management/mock/blob/main/mock/docs/mock.1#L417) setting [conditionals build options](https://rpm-software-management.github.io/rpm/manual/conditionalbuilds.html). AFAIK no one requested this functionality from us yet (not used anymore?). Nevertheless, I just wanted to mention the feature here.

No need to change anything in the research doc. Let's keep the scope down.

Reply:
I'd be more afraid about shared sources between packages, e.g. helpers, utility libraries.

_edit_: never mind, there's `package_paths`